aladdin.sh

```diff
@@ -121,112 +115,25 @@ function check_and_handle_init() {
         if "$MANAGE_SOFTWARE_DEPENDENCIES"; then
             "$SCRIPT_DIR"/infra_k8s_check.sh
         fi
-        enter_minikube_env
+        check_or_start_k3d
     fi
 }
 
-function set_minikube_config(){
-    if "$MANAGE_MINIKUBE"; then
-        minikube config set kubernetes-version v$KUBERNETES_VERSION > /dev/null
-
-        for key in vm_driver memory disk_size cpus; do
-            local minikube_key=$(tr _ - <<< "$key")  # e.g., driver
-            local default_var="DEFAULT_MINIKUBE_$(tr a-z A-Z <<< "$key")"  # e.g., DEFAULT_MINIKUBE_VM_DRIVER
-
-            local value=$(aladdin config get "minikube.$key" "${!default_var:-}")
-
-            if test -n "$value"; then
-                minikube config set "$minikube_key" "$value" > /dev/null
-            fi
-        done
-    fi
+function _start_k3d() {
+    # start k3d cluster
+    k3d cluster create LOCAL --api-port "$K3D_API_PORT" -p "$K3D_SERVICE_PORT:80@loadbalancer" \
+        --k3s-server-arg "--tls-san=host.docker.internal" --image "rancher/k3s:v$KUBERNETES_VERSION-k3s1"
```

Comment:
tls-san argument needed to that we don't get certificate issues when hitting the kubernetes api server from within the docker container